### PR TITLE
Update content about the InfluxQL shell in 2.4 migration guide

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.4/upgrade/v1-to-v2/automatic-upgrade.md [hunk 1]
```diff
@@ -174,7 +175,20 @@ InfluxDB {{< current-version >}} doesn't support [in-memory (`inmem`) indexing](
...
+To start an InfluxQL shell:
+
+1. Download and install in the [`influx` CLI](/influxdb/v2.4/tools/influx-cli/).
+2. Set up your [`influx` CLI authentication credentials](/influxdb/v2.4/tools/influx-cli/#set-up-the-influx-cli).
+3. Run the `influx v1 shell` command.
+
+For more information see:
+
+- [Use the InfluxQL shell](/influxdb/v2.4/tools/influxql-shell/)
+- [Query data with InfluxQL](/influxdb/v2.4/query-data/influxql/)
```
Comment (kelseiv) [inline content/influxdb/v2.4/upgrade/v1-to-v2/automatic-upgrade.md:188]: Would it make sense to include this info for manual and docker upgrades as well?

If yes, perhaps in each upgrade doc, we say something along the lines of:
After upgrading, you have the option to use an interactive shell to query data in InfluxDB. For information about querying InfluxQL, see how to use the InfluxQL shell. For information about querying Flux, see...

Thought about alternative of adding to "Upgrade from InfluxDB 1.x to 2.4" and occurs that folks may click one of the choices manual, automatic, etc, and then we'd need to link from procedure back to top-level, which felt a bit circular

Comment (sanderson) [reply content/influxdb/v2.4/upgrade/v1-to-v2/automatic-upgrade.md:188]: Added InfluxQL info to the other upgrade docs.

File: content/influxdb/v2.4/upgrade/v1-to-v2/docker.md [hunk 2]
```diff
@@ -249,3 +250,41 @@ docker run -p 8086:8086 \
...
+## Use new InfluxDB tools
+
+Once upgraded, use InfluxDB {{< current-version >}} tools to work with your time series data.
+
+- [Migrate continuous queries to tasks](#migrate-continuous-queries-to-tasks)
+- [Use the interactive InfluxQL shell](#use-the-interactive-influxql-shell)
+
+### Migrate continuous queries to tasks
+
+InfluxDB {{< current-version >}} replaces continuous queries **tasks**.
```
Comment (kelseiv) [inline content/influxdb/v2.4/upgrade/v1-to-v2/docker.md:263]:
```suggestion
InfluxDB {{< current-version >}} replaces continuous queries with **tasks**.
```

File: content/influxdb/v2.4/upgrade/v1-to-v2/docker.md [hunk 3]
```diff
@@ -249,3 +250,41 @@ docker run -p 8086:8086 \
...
+[Migrate continuous queries to tasks](/influxdb/v2.4/upgrade/v1-to-v2/migrate-cqs/).
+
+### Use the interactive InfluxQL shell
+
+The InfluxDB {{< current-version >}} `influx` CLI includes an interactive **InfluxQL shell** for executing InfluxQL queries.
+The InfluxDB {{< current-version >}} Docker image includes the `influx` CLI.
+
+To start an InfluxQL shell:
+
+1. Console into your InfluxDB container:
```
Comment (kelseiv) [inline content/influxdb/v2.4/upgrade/v1-to-v2/docker.md:275]: Log into? Run your InfluxDB container in Docker?

Comment (kelseiv) [reply content/influxdb/v2.4/upgrade/v1-to-v2/docker.md:275]: verifying re https://developers.google.com/style/word-list#letter-c

File: content/influxdb/v2.4/upgrade/v1-to-v2/docker.md [hunk 4]
```diff
@@ -249,3 +250,41 @@ docker run -p 8086:8086 \
...
+
+    ```sh
+    docker exec -it influxdb /bin/bash
+    ```
+
+2. Set up your [`influx` CLI authentication credentials](/influxdb/v2.4/tools/influx-cli/#set-up-the-influx-cli).
+3. Run the `influx v1 shell` command.
+
+For more information see:
```
Comment (kelseiv) [inline content/influxdb/v2.4/upgrade/v1-to-v2/docker.md:284]:
```suggestion
For more information, see:
```

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.4/upgrade/v1-to-v2/docker.md (hunk 3 above)
Comment (sanderson) [inline content/influxdb/v2.4/upgrade/v1-to-v2/docker.md:275]:
```suggestion
1. Start a bash session in your InfluxDB container:
```

**Pull request merged**